db/migrate/20230925094135_change_git_username_to_github_handle.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeGitUsernameToGithubHandle < ActiveRecord::Migration[6.1]
+  def change
+    rename_column :users, :git_username, :github_handle
+  end
```

Comment:
since we haven't deployed the Database migration to the Production server, we should modify the original add_column migration to have the new name.

the rename_column migration can be removed.

On your local make sure to 
- first rollback the 2 migrations
- delete the second (rename_column) migration
- edit the first (add_column) migration with the new column name
- run db migrate